src/Model/Entity/BussinessObject/Introduction.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Friendica\Model\Entity;
```

Comment:
Shouldn't it be `Friendica\Model\Entity\BusinessObject` per the folder structure?

Reply:
Yep